browser/files_test.py

```diff
@@ -0,0 +1,22 @@
+"""Test for File classes"""
+
+import pytest
+import numpy as np
+
+from files import CalibanFile
+
+
+def test_init(file_):
```

Comment:
Consider adding a test for the `create_cell_info` method, though it may overlap with the current tests.